src/main/java/com/google/sps/data/DeleteAllFromDatastore.java

```diff
@@ -0,0 +1,28 @@
+package com.google.sps.data;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+
+// This class is to be used by the other three servlets to delete entities from datastore.
+public class DeleteAllFromDatastore {
+
+  /*
+   *  @param query to get results 
+   *  @param datastore entities are stored here
+   *  @param kind the kind of the entity
+  */
+  public static void deleteResultsOfQueryFromDatastore(Query query, DatastoreService datastore, String kind) {
```

Comment:
type/servlet type might be better variable names than "kind".

Reply:
So, I gave that name since in the documentation, "Article", "Game", and "Video" are the kind of the entity. https://screenshot.googleplex.com/7d9f6e6b-ce69-45b7-ad80-612e5616b5a2.png

Reply:
Oh okay, that looks good to me.